src/org/usfirst/frc/team2706/robot/Robot.java

```diff
@@ -107,7 +111,9 @@ public void robotInit() {
      * This function is called once each time the robot enters Disabled mode. You can use it to
      * reset any subsystem information you want to clear when the robot is disabled.
      */
-    public void disabledInit() {}
+    public void disabledInit() {
+        blingSystem.clear(); // S.V.P leave this here. 
```

Comment:
That command doesn't cancel, it tells the arduino to clear the LED strip. Otherwise, the pattern will stay on. I guess we could leave it to the command to do that in the `end()` and `interrupted()` commands. I'll change that.